config/pmd.xml

```diff
@@ -34,7 +34,11 @@
     <!-- we use class comments as source for xdoc files, so content is big and that is by design -->
     <exclude name="CommentSize"/>
   </rule>
-  <rule ref="rulesets/java/comments.xml/CommentRequired" />
+  <rule ref="rulesets/java/comments.xml/CommentRequired">
+    <properties>
+      <property name="violationSuppressXPath" value="//Annotation/MarkerAnnotation//Name[@Image='Override']"/>
```

Comment:
Please explain this change.

Reply:
Without this pmd would force us to add javadoc on `@Override` method. I copied this settings from checkstyle repo.

Reply:
Ok, thanks.
This can be ignored.